src/Classes/RichTextViewController.swift

```diff
@@ -473,108 +473,130 @@ public class RichTextViewController: UIViewController {
                 }
             }
         }
-
+        
         if range.location != textView.selectedRange.location || range.length != textView.selectedRange.length {
             textView.selectedRange = range
         }
     }
-
+    
     // MARK: Font Adjustments
-
+    
     private func applyFontAttribute(font: UIFont) {
         guard let attributedString = textView.attributedText else { return }
-
+        
         let attributedText = NSMutableAttributedString(attributedString: attributedString)
-
+        
         attributedText.beginEditing()
         attributedText.enumerateAttributesInRange(textView.selectedRange, options: []) { _, range, _ in
             attributedText.addAttribute(NSFontAttributeName, value: font, range: range)
         }
         attributedText.endEditing()
-
+        
         textView.attributedText = attributedText
     }
-
+    
+    private func removeFormattingFromListLeadsInRange(range: NSRange) {
+        guard range.length > 0 else { return }
+        
+        var listHeadRegex: NSRegularExpression?
+        
+        do {
+            listHeadRegex = try NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines)
+        } catch {
+            print("Failed to remove formatting")
+        }
+        
+        listHeadRegex?.matchesInString(textView.text, options: [], range: range).forEach { match in
+            guard let regularFont = self.regularFont else { return }
```

Comment:
Maybe even as part of the guard on line 499 since nothing useful will happen if it fails.